tests/Zend/Filter/HtmlEntitiesTest.php

```diff
@@ -214,7 +214,7 @@ public function testQuoteStyleQuotesEncodeNone()
      */
     public function testCorrectsForEncodingMismatch()
     {
-        if (PHP_VERSION_ID >= 50400) {
+        if (PHP_VERSION_ID >= 70000 && PHP_VERSION_ID < 70100) {
```

Comment:
is there any chance that changes in this file could affect the failing Logger error on PHP 5.4? 🙈